modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -225,6 +225,28 @@
         end
       end
 
+      context 'pdf verification alternate signer' do
+        let(:supplemental_claim) { create(:supplemental_claim_alt_signer, evidence_submission_indicated: true, created_at: '2021-02-03T14:15:16Z') }
+
+        it 'generates the expected pdf' do
+          generated_pdf = described_class.new(supplemental_claim, pdf_version: 'v2').generate
+          expected_pdf = fixture_filepath('expected_200995_alternate_signer.pdf', version: 'v2')
+          expect(generated_pdf).to match_pdf(expected_pdf)
+          File.delete(generated_pdf) if File.exist?(generated_pdf)
+        end
+      end
+
+      context 'pdf verification alternate signer overflow' do
+        let(:supplemental_claim) { create(:supplemental_claim_alt_signer_overflow, evidence_submission_indicated: true, created_at: '2021-02-03T14:15:16Z') }
+
+        it 'generates the expected pdf' do
+          generated_pdf = described_class.new(supplemental_claim, pdf_version: 'v2').generate
+          expected_pdf = fixture_filepath('expected_200995_alternate_signer_overflow.pdf', version: 'v2')
```

Comment:
Remind me to talk to you and @caseywilliams about how we go about doing content validation across our pdfs. I open this convo up every once in a while just to make sure the team is good with how we do this sort of thing. I.e. is using gold masters rather than inspecting individual pages of the pdf for explicit content our preferred method of validation. Just something to noodle over and let me know if you've got any opinions :)